README.md

```diff
@@ -68,12 +68,20 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- You can use `pip`:
+
+    pip install spyder
+
+- Or install one scientific Python distributions that come with Spyder:
   1. [Anaconda](http://continuum.io/downloads)
   2. [WinPython](https://winpython.github.io/)
-  3. [Python(x,y)](http://pythonxy.googlecode.com)
-- Using one of our executable installers, which can be found
-  [here](https://github.com/spyder-ide/spyder/releases).
+  3. [Python(x,y)](http://python-xy.github.io/)
+
+We used to produce [executable installers](https://github.com/spyder-ide/spyder/releases),
+but they were missing required dependencies and with Spyder 3.x
+we've decided to drop them until somebody comes with a solution
+for more friendly installers (see #2648).
```

Comment:
I am still trying to make it run with bootstrap script, really, and it is failure after failure. =)